src/playback/internal/playbackuiactions.cpp

```diff
@@ -108,14 +108,14 @@ const UiActionList PlaybackUiActions::m_settingsActions = {
              IconCode::Code::PAN_SCORE,
              Checkable::Yes
              ),
-    UiAction("countin",
-             mu::context::UiCtxAny,
-             mu::context::CTX_ANY,
-             TranslatableString("action", "Enable count-in when playing"),
-             TranslatableString("action", "Enable count-in when playing"),
-             IconCode::Code::COUNT_IN,
-             Checkable::Yes
-             ),
+//    UiAction("countin",                                      // See #14807
```

Comment:
You should keep only this change. All changes in other files should be excluded from this PR